ui/shared/ConfirmationDialog.qml

```diff
@@ -26,6 +27,7 @@ ModalPopup {
 
     Text {
         text: confirmationDialog.confirmationText
+        color: Style.current.textColor
```

Comment:
This isn't needed if you use StyledText instead
```suggestion
```

Reply:
Dang! Shoulda noticed! Fixed!